lib/src/rlp.nr

```diff
@@ -114,9 +117,12 @@ pub fn decode_len<N>(arr: [u8; N]) -> RLP_Header
             strict_if(
                 prefix_p2,
                 (prefix - 0x80) as Field,
```

Comment:
Running `nargo test` in version `0.19.4` fails in this line with error:
```rust
error: Assertion failed: 'attempt to subtract with overflow'
    ┌─ /Users/user/projects/noir-trie-proofs/lib/src/rlp.nr:119:18
    │
119 │                 (prefix - 0x80) as Field,
    │                  -------------
    │
    = Call stack:
      1. /Users/akonior/projects/zk/github/forks/noir-trie-proofs/lib/src/rlp.nr:257:31
      2. /Users/akonior/projects/zk/github/forks/noir-trie-proofs/lib/src/rlp.nr:176:88
      3. /Users/akonior/projects/zk/github/forks/noir-trie-proofs/lib/src/rlp.nr:119:18
  ```
Probably this should also be replaced with `wrapping_sub`

Reply:
Thanks for the good catch!